### ISSUE TITLE
Feature Request - Network IP Widget

### ISSUE THREAD

**Issue by PMillard5000** [tjmarkham/win10widgets#56]
Hi there,

Love the widgets that you have produced.
I was just wondering if it would be possible to create a widget that would list your LAN, WIFI and Public IP (Both IPv4 and IPv6) addresses.
As a techy jumping on and off different networks all the time would be really useful.

Many thanks

**Comment by joemurrell** [comment]
Seconded.

**Comment by joshkmartinez** [comment]
Yes please, I would love this!

**Comment by jintadhee** [comment]
I would love this too.

**Comment by maskedmouse** [comment]
I was looking for this as well. Would be quite handy to see in one glance what the IP is. The request is from quite some time ago but hopefully it'll be implemented at some point.

**Comment by turexy** [comment]
Yes that would be a lovely feature to see current public IP especially when more and more people are using a variety of  VPNs services to private their identity.
